front/app/components/IdeaButton/index.tsx

```diff
@@ -323,16 +323,22 @@ const IdeaButton = memo<Props & InjectedIntlProps>(
                   disabled={!enabled}
                   ariaDisabled={false}
                 >
-                  <FormattedMessage
-                    {...getInputTermMessage(inputTerm, {
-                      idea: messages.submitYourIdea,
-                      option: messages.addAnOption,
-                      project: messages.addAProject,
-                      question: messages.addAQuestion,
-                      issue: messages.submitAnIssue,
-                      contribution: messages.addAContribution,
-                    })}
-                  />
+                  {project.attributes.participation_method === 'ideation' && (
+                    <FormattedMessage
+                      {...getInputTermMessage(inputTerm, {
+                        idea: messages.submitYourIdea,
+                        option: messages.addAnOption,
+                        project: messages.addAProject,
+                        question: messages.addAQuestion,
+                        issue: messages.submitAnIssue,
+                        contribution: messages.addAContribution,
+                      })}
+                    />
+                  )}
+                  {project.attributes.participation_method ===
+                    'native_survey' && (
```

Comment:
In the future we could maybe use a more general check or new property, but I think at some point we'll need a check to make sure we are using the correct copy for each input type (i.e. if _native survey_, then...)?